qdrant-landing/content/articles/qdrant-1.3.x.md

```diff
@@ -0,0 +1,202 @@
+        // Options for specifying what to bring from the payload 
+        // of the looked up point, true by default
+        "with_payload": ["title", "text"],
+        // Options for specifying what to bring from the vector(s) 
+        // of the looked up point, false by default
+        "with_vector: false,
+    }
+}
+```
+
+```python
+client.search_groups(
+    collection_name="chunks",
+
+    # Same as in the regular search() API
+    vector=[1.1],
+    ...,
+
+    # Grouping parameters
+    group_by="document_id", # Path of the field to group by
+    limit=2,                # Max amount of groups
+    group_size=2,           # Max amount of points per group
+
+    # Lookup parameters
+    with_lookup=models.WithLookup(
+        # Name of the collection to look up points in
+        collection_name="documents",
+
+        # Options for specifying what to bring from the payload 
+        # of the looked up point, True by default
+        with_payload=["title", "text"]
+
+        # Options for specifying what to bring from the vector(s) 
+        # of the looked up point, False by default
+        with_vector=False, 
+    )
+)
+```
+
+### Qdrant web user interface
+
+We are excited to announce a more user-friendly way to organize and work with your collections inside of Qdrant. Our dashboard's design is simple, but very intuitive and easy to access.
+
+Try it out now! If you have Docker running, you can [quickstart Qdrant](https://qdrant.tech/documentation/quick-start/) and access the Dashboard locally from [http://localhost:6333/dashboard](http://localhost:6333/dashboard). You should see this simple access point to Qdrant:
+
+![Qdrant Web UI](/articles_data/qdrant-1.3.x/web-ui.png)
+
+### Temporary directory for Snapshots
+
+Currently, temporary snapshot files are created inside the `/storage` directory. Oftentimes `/storage` is a network-mounted disk. Therefore, we found this method suboptimal because `/storage` is limited in disk size and also because writing data to it may affect disk performance as it consumes bandwidth. This new feature allows you to specify a different directory on another disk that is faster. We expect this feature to significantly optimize cloud performance.
+
+To change it, access `config.yaml` and set `storage.temp_path` to another directory location.
+
+## Important changes
+
+The latest release focuses not only on the new features but also introduces some changes making 
+Qdrant even more reliable.
+
+### Optimizing group requests
+
+Internally, we improved the way grouping API requests are handled. Specifically, `is_empty` was not using the index when it was called, so it had to deserialize the whole payload to see if the key had values or not. Our new update makes sure to check the index first, before confirming with the payload if it is actually `empty`/`null`, so these changes improve performance only when the negated condition is true (e.g. it improves when the field is not empty).
```

Comment:
Maybe separate the fact that we optimized `is_empty` and that it consequently improved `group by`